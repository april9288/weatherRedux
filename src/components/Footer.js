import React from 'react';
import PropTypes from 'prop-types';
import AppBar from '@material-ui/core/AppBar';
import Grid from '@material-ui/core/Grid';
import Divider from '@material-ui/core/Divider';
import { withStyles } from '@material-ui/core/styles';

const styles = theme => ({
  root: {
    width: '100%',
    alignItems: "center!important",
    display: "flex!important",
    justifyContent: "center!important",
  },
  footerContainer:{
    padding: "3rem 0 1rem 0",
    fontWeight: 600
  }
});

const Footer = ({ classes }) => {

  return (
    <div className={classes.root}>
      <AppBar position="static">
        <Grid container className={classes.footerContainer}>

            <Grid item xs={12} sm={12} className={classes.bottomGrid}>
              <a target="_blank" href="https://github.com/april9288/" sytle={{textDecoration: "none"}}><i className="fab fa-github-square footerIcons"></i></a>
              <a target="_blank" href="https://www.linkedin.com/in/jongho-kim-b05618170/" sytle={{textDecoration: "none"}}><i className="fab fa-linkedin footerIcons"></i></a>
            </Grid>

            <Divider />

            <Grid item xs={12} sm={12} className={classes.bottomGrid}>
              <p className={classes.title}>
                Made with Love By James Jongho Kim
              </p>
            </Grid>

        </Grid>
      </AppBar>
    </div>
  );
}

Footer.propTypes = {
  classes: PropTypes.object.isRequired,
};

export default withStyles(styles)(Footer);


