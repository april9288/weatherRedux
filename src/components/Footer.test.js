import { shallow } from 'enzyme'
import React from 'react'
import Footer from './Footer'

it('expects to render Footer comp', ()=> {
	expect(shallow(<Footer />)).toMatchSnapshot()
})