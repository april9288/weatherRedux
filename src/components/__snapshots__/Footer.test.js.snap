// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expects to render Footer comp 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <WithStyles(Footer) />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "classes": Object {
        "footerContainer": "Footer-footerContainer-2",
        "root": "Footer-root-1",
      },
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "classes": Object {
          "footerContainer": "Footer-footerContainer-2",
          "root": "Footer-root-1",
        },
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
